server/src/main/java/io/druid/segment/realtime/firehose/PredicateFirehose.java

```diff
@@ -55,6 +60,11 @@ public boolean hasMore()
         savedInputRow = row;
         return true;
       }
+      // Do not silently discard the rows
+      if (ignored % ignoredThreshold == 0) {
```

Comment:
it will warn as soon as first row is ignored and then for each 5000th ignored row. The purpose of the PR is to have *some* warning in the logs than to silently ignore it for which I think having exponential backoff strategy is more than necessary.